apps/create_covering_animation.py

```diff
@@ -51,6 +51,10 @@ def data_generator(node_data, edges, sample_points, dstep):
 if __name__ == '__main__':
 
     filename = sys.argv[1]
+    if len(sys.argv) > 2:
+        output = sys.argv[2]
+    else:
+        output = 'test.mp4'
```

Comment:
Tha ftiaksw kanoniko argparse gia to app. Opote to afhnw gia na kaneis th douleia sou kai sto mellon tha kapws ets:
--out=blablah.mp4

Reply:
Works as a heck :p